docs/ModelZoo.md

```diff
@@ -0,0 +1,950 @@
+# NNCF Compressed Model Zoo
+
+Here we present the results achieved using our sample scripts, example patches to third-party repositories and NNCF configuration files.
+
```

Comment:
Let's add a note that PTQ accuracy and performance results for variety of networks can be found in https://docs.openvino.ai/latest/openvino_docs_performance_benchmarks.html

Reply:
Done